lib/master.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file In order to load huge volumes of polygon data into memory without
+ *    breaking Node (due to its low heap-size limits), the package forks a
+ *    worker process per polygon layer/shapefile. This module contains
+ *    functions for initializing them and searching them.
+ */
+
+'use strict';
+
+var childProcess = require( 'child_process' );
+var peliasConfig = require( 'pelias-config' );
+var path = require( 'path' );
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'admin-lookup:master' );
+
+/**
+ * For every desired administrative layer, specifies its name, the partial path
+ * of the corresponding Quattroshapes file, and the properties to extract from
+ * it.
+ */
+var quattroAdminLevels = [
```

Comment:
I'm not sure what the use-case for putting this in `pelias-config` is -- what would you want to change? Also, why did you have difficulty running the tests? If it was because of the Quattroshapes path, `pelias-config` specifies a default that you can override in your custom config.